cypress/integration/inventory/Item-status-date-updates.spec.js

```diff
@@ -0,0 +1,440 @@
+              .then(rules => {
+                rulesDefaultString = rules.rulesAsText;
+              });
+          })
+          .then(() => {
+            const requestPolicyId = Cypress.env(CY_ENV.REQUEST_POLICY)[0].id;
+            const noticePolicyId = Cypress.env(CY_ENV.NOTICE_POLICY)[0].id;
+            const overdueFinePolicyId = Cypress.env(CY_ENV.OVERDUE_FINE_POLICY)[0].id;
+            const lostItemFeesPolicyId = Cypress.env(CY_ENV.LOST_ITEM_FEES_POLICY)[0].id;
+            const loanPolicyId = Cypress.env(CY_ENV.LOAN_POLICIES)[0].id;
+            const newRule = `\ng ${user.patronGroup} + m ${orderLine.physical.materialType}: l ${loanPolicyId} r ${requestPolicyId} n ${noticePolicyId} o ${overdueFinePolicyId} i ${lostItemFeesPolicyId}`;
+
+            cy.updateCirculationRules({
+              rulesAsText: rulesDefaultString + newRule,
+            });
+          });
+        cy.getUsers({ limit: 1, query: `"personal.lastName"="${user.username}" and "active"="true"` })
+          .then((users) => {
+            userBarcode = users[0].barcode;
+          });
+      });
+
+    cy.createTempUser([
+      permissions.checkoutAll.gui,
+      permissions.requestsAll.gui,
+    ])
+      .then(userProperties => {
+        userForDeliveryRequest = userProperties;
+        cy.getUsers({ limit: 1, query: `"username"="${userForDeliveryRequest.username}"` })
+          .then((users) => {
+            userForDeliveryRequest.barcode = users[0].barcode;
+            userRequestPreferences.userId = users[0].id;
+            cy.getAddressTypesApi({ limit: 1 }).then(addressTypes => {
+              userRequestPreferences.defaultDeliveryAddressTypeId = addressTypes[0].id;
+            })
+              .then(() => {
+                cy.updateUser({
+                  ...users[0],
+                  personal: {
+                    lastName: '',
+                    addresses: [{ city: 'New York',
+                      addressTypeId: userRequestPreferences.defaultDeliveryAddressTypeId,
+                      countryId: 'US' }]
+                  }
+                });
+                cy.createUserRequestPreferencesApi(userRequestPreferences);
+              });
+          });
+      });
+  });
+
+  afterEach(() => {
+    cy.getInstance({ limit: 1, expandAll: true, query: `"items.barcode"=="${userItemBarcode}"` })
+      .then((instance) => {
+        instance.items.forEach((item) => {
+          cy.deleteItem(item.id);
+        });
+        cy.deleteHoldingRecord(instance.holdings[0].id);
+        cy.deleteInstanceApi(instance.id);
+      });
+    cy.deleteOrderApi(order.id);
+    cy.deleteUser(user.userId);
+    cy.deleteServicePoint(notEffectiveLocationServicePoint.body.id);
+    // cy.deleteServicePoint(effectiveLocationServicePoint.body.id);
+    // Requests.getRequestApi({ limit: 1, query: `"item.barcode"=="${userItemBarcode}"` })
+    // .then(request => {
+    // Requests.deleteRequestApi(request.id);
+    // });
```

Comment:
Why this code commented?
Uncomment or delete it please)

Reply:
Done